spec/index.html

```diff
@@ -423,5 +440,80 @@ <h3>Examples</h3>
+
+      <div class="practice">
+        <p class="practicedesc">
+          <span id="conceal-contexts" class="practicelab">Use a default context</span>
+        </p>
+
+        If most, or all, of a user's documents are based on one particular context, try to make it the default in order to rescue the user from copy-pasting the same technical incantation from one document to another.
+      </div>
+
+      <p>For instance, according to [[json-ld-api]], `expand()` function of a JSON-LD processor accepts an `expandContext` argument which can be used to provide a default system context.</p>
+
+      <div class="practice">
+        <p class="practicedesc">
+          <span id="alias-keywords" class="practicelab">Alias JSON-LD keywords</span>
+
+          If possible, map JSON-LD keywords containing the `@` character to keywords that do not contain it.
+        </p>
+      </div>
+
+      <p>The `@` character is reserved as per [[YAML]] specification and thus requires quoting, as in the following example:</p>
+
+      <aside class="example" data-transform="updateExample"
+             title="Example YAML-LD document without Convenience Context">
+        <pre class="yaml" data-include="code/without-convenience-context.yaml" data-include-format="text"></pre>
+      </aside>
+
+      <p>The need to quote these keywords has to be learnt, and introduces one more little irregularity to the document author's life. Besides, on most keyboard layouts typing quotes will require `Shift` — which albeit slightly but does reduce typing speed.</p>
+
+      <p>In order to avoid this, the context might introduce custom mappings for the necessary keywords. For instance, [[schema-org]] context redefines `@id` as just `id` — which seems to be much more convenient to type, and no more difficult to remember.</p>
+
+       <div class="practice">
+          <p class="practicedesc">
+            <span id="convenience-context" class="practicelab">Use YAML-LD Convenience Context</span>
+            <p>YAML-LD users may use a JSON-LD context provided as part of this specification, henceforth known as the <dfn>convenience
+          context</dfn>, which defines a standardized mapping of every `@`-keyword to a `$`-keyword, except `@context`.</p>
+         </p>
+       </div>
+
+      <aside class="example" data-transform="updateExample" title="Convenience Context">
+        <p>Code of the <a>convenience context</a>. Available as: <a href="https://yaml-ld.dev/context">https://yaml-ld.dev/context</a>.
+        </p>
+        <pre class="json" data-include="code/context.json" data-include-format="text"></pre>
```

Comment:
It might be worth inlining these to avoid having included files dangle. Also, PR preview doesn't seem to handle these well.

Reply:
Indeed… I hoped that including files would help editing experience, to provide syntax highlighting in editors and to be able to perhaps autotest the included files. What is your view on these potential advantages?

Reply:
We did all JSON-LD examples inline, using some helper JS to avoid escaping. I implemented a CI mechanism to validate all the examples, and we should do something like that here as well; I can certainly help facilitate this if you can stand some Ruby :).

ReSpec has had some issues with referenced files that have bitten me before when using the spec-prod GH Action, but it is a fair thing to use. Too bad the PR-preview rendering seems broken.